catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -147,13 +155,42 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
       query.setFacetMinCount(textFacetProp.getMinFacetCount());
     }
 
-    long totalHits;
-    List<Result> results = new ArrayList<>();
-    Map<String, Serializable> responseProps = new HashMap<>();
+    Serializable suggestQueryProp = request.getPropertyValue(SUGGESTION_REQUEST_KEY);
+
+    if (suggestQueryProp instanceof String) {
+      query = new SolrQuery();
+      query.setRequestHandler("/suggest");
+      query.setParam(SUGGEST_Q, (String) suggestQueryProp);
+      query.setParam(SUGGEST_CONTEXT_FILTER_QUERY, "gazetteer");
```

Comment:
Since we are now using this in SOLR, we might want to move this constant 
https://github.com/codice/ddf/blob/master/catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GeoCodingConstants.java#L46
here
https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-api/src/main/java/ddf/catalog/Constants.java
and use it